### DISCUSSION TITLE
Run separation on iOS offline

### DISCUSSION THREAD

**Discussion by day-dreaming-guy** [CarlGao4/Demucs-Gui#82] [General]
Hey! Is it possible to run this on an iPhone? I was curious if it’s supported, as it’s running on a Mac with an Apple chip.

**Comment by CarlGao4** [comment]
Actually, no. Maybe you can reach this by rewriting it with C++ using torchlib, or compiling PyTorch dylib and Python for iOS then run the python script

**Comment by day-dreaming-guy** [comment]
Ah, I got it, thanks! I'll need to use an API then..
